### PR TITLE
mvar: add dependency on opam-installer

### PR DESCRIPTION
**Pull request by damiendoligez**

### CODE REVIEW

File: packages/mvar/mvar.1.0.0/opam [hunk 1]
```diff
@@ -18,6 +18,7 @@ remove: [
 ]
 depends: [
   "jbuilder" {build}
+  "opam-installer"
```
Comment (samoht) [inline packages/mvar/mvar.1.0.0/opam]: I guess it's a `{build}` dependency too, right?

Comment (damiendoligez) [reply packages/mvar/mvar.1.0.0/opam]: That's a tough question. It's not used at build-time, it's used at install-time. What does the `{build}` modifier mean exactly? OPAM documentation doesn't say.

Comment (samoht) [reply packages/mvar/mvar.1.0.0/opam]: It just means that reinstalling `opam-installer` will not trigger a recompilation of `mvar`. I guess it works for an intall-time dependency too.

**Comment by damiendoligez** [comment]
added `{build}` annotation

**Comment by hannesm** [comment]
thanks, I reported the travis CI failure (of re in build-test on 4.06.0) at https://github.com/ocaml/ocaml-re/issues/153

**Pull request merged**